src/app/pages/directory-service/directory-services.elements.ts

```diff
@@ -9,13 +9,9 @@ export const directoryServicesElements = {
       anchor: 'directory-services',
     },
     configureActiveDirectory: {
-      hierarchy: [T('Configure Active Directory')],
-      synonyms: [T('Active Directory')],
+      hierarchy: [T('Configure Directory Services')],
+      synonyms: [T('Active Directory'), 'AD', 'LDAP', 'OpenLDAP', 'IPA', 'FreeIPA', 'DS'],
       anchor: 'configure-active-directory',
```

Comment:
So what happens if you have AD configured and then search for these terms?

I also notice the anchor has active-directory in the name and the key here is configureActiveDirectory